prime-router/docs/unused_dependency_check.md

```diff
@@ -0,0 +1,11 @@
+# Unuused dependency check
```

Comment:
Maybe one documentation can just point to the other (just a link in the doc is fine)... just to avoid having to maintain it both places?